pallet-tft-bridge/src/lib.rs

```diff
@@ -34,6 +29,9 @@ pub trait Config: system::Config {
 
 	/// Handler for the unbalanced decrement when slashing (burning collateral)
 	type Burn: OnUnbalanced<NegativeImbalanceOf<Self>>;
+
+	/// Origin for sudo extrinsics
```

Comment:
sudo is just a pallet to dispatch calls as the Root origin, they are not sudo extrinsics